File: 2114_max_number_of_words_from_sentences/find_max_words.js
// JS: https://leetcode.com/problems/maximum-number-of-words-found-in-sentences/submissions/1188888334/
// TS: https://leetcode.com/problems/maximum-number-of-words-found-in-sentences/submissions/1188866846/
function find_words(sentences) {
    let results = [];

    for(let i = 0; i < sentences.length; i++) {
        results.push(sentences.split(" ").length);
    }

    return Math.max(...results)
}

File: 2114_max_number_of_words_from_sentences/find_max_words.py
# Py3: https://leetcode.com/problems/maximum-number-of-words-found-in-sentences/submissions/1188900697/
from typing import List

def find_words(senetences: List[str]):
    results = []

    for s in senetences:
        results.append(len(s.split()))

    return 0 if len(results) == 0 else results[0] if len(results) else max(*results)

print(find_words(["rjtrogrmvpzm vezxyl irqufo"]))
print(find_words([]))